sites/public/.env.template

```diff
@@ -24,7 +24,7 @@ MAINTENANCE_WINDOW=
 GTM_KEY=GTM-KF22FJP
 
 # feature toggles
-SHOW_LOTTERY=TRUE
+SHOW_LOTTERY_APPS=TRUE
```

Comment:
Could we name this more generically around something about the public site? We will also use this toggle in Doorway on the public site to toggle the help center content about lotteries

Reply:
bump